mobile/src/employee-details/employee-details.tsx

```diff
@@ -82,14 +105,18 @@ export class EmployeeDetailsImpl extends Component<EmployeeDetailsProps & Employ
     public componentDidMount() {
```

Comment:
not sure about componentDidMount at all... how does profile work?

Reply:
it works, since the component won't be rendered with empty employee (there is a check in ProfileScreen)